### PR TITLE
feat: add hover popup to centroids and project features

### PR DESCRIPTION
**Pull request by steveoh**
refs #16

### CODE REVIEW

File: src/components/MapContainer.tsx [hunk 1]
```diff
@@ -115,23 +118,27 @@ export const MapContainer = ({ configuration }: { configuration: 'search' | 'edi
...
+        addLayers(referenceLayers.concat(operationalLayers.current));
       }
     }
-    setMapView(mapView.current!);
   }, [isReady, mapView, addLayers, setMapView, configuration]);
 
+  // if (!isReady) {
+  //   return <BusyBar busy={true} />;
+  // }
+
```
Comment (stdavis) [inline src/components/MapContainer.tsx:135]: Did you mean to leave this in here?

File: src/mapLayers.ts [hunk 2]
```diff
@@ -1005,8 +1005,8 @@ export const stewardship = new FeatureLayer({
 export const points = new FeatureLayer({
   title: 'Points',
   id: 'feature-point',
-  url: 'https://wrimaps.utah.gov/arcgis/rest/services/WRI/Features/MapServer/0',
-  fields: ['StatusDescription'],
+  url: 'https://wrimaps.at.utah.gov/arcgis/rest/services/WRI_AT/Features/MapServer/0',
```
Comment (stdavis) [inline src/mapLayers.ts:1008]: I assume that this base path will be moved to an env var or something similar at some point?

**Review by stdavis — APPROVED** [review]
This looks great. I like the idea of using the DOM api's for moving the tooltip around rather than rerendering the component.

I also had the thought today that perhaps a hover popup could be configured as part of the feature service? Or maybe that's just an AGOL thing. Either way, I think that this solution is great.

File: src/mapLayers.ts (hunk 2 above)
Comment (steveoh) [reply src/mapLayers.ts:1008]: Yes, I'm waiting to see how things play out with the DTS integration.

**Comment by steveoh** [comment]
> I also had the thought today that perhaps a hover popup could be configured as part of the feature service.

I wasn't able to find this. Is this a thing?

**Comment by stdavis** [comment]
> I wasn't able to find this. Is this a thing?

I'm not sure. Maybe it's an AGOL thing, not stand-alone server.

**Review by stdavis — APPROVED** [review]

**Pull request merged**